VestaRestPackage/app_objects.py

```diff
@@ -11,6 +11,13 @@
 
 from . import default_configuration
 from . import celery_init
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+
+sentry_sdk.init(
+    integrations=[FlaskIntegration()]
```

Comment:
In the init documentation, the parameter `dsn` is set. 
Is there an other way the `dsn` value is set?

Reply:
Sentry SDK is designed to look for SENTRY_DSN environment variable: https://docs.sentry.io/clients/node/config/#environment-variables